### ISSUE TITLE
How to enable the Zero-Copy Video Capture flag setting in electron?

### ISSUE THREAD

**Issue by xiaoxiong83** [electron/electron-quick-start-typescript#270]
In chrome browser, I can enable this flag in the chrome://flags setting page, but in electron, how to enable this flag?

**Comment by kentezrabetita** [comment]
You can try to enable this feature using the `app.commandLine.appendSwitch` method, like this:

```javascript
const { app } = require('electron')

app.commandLine.appendSwitch('enable-zero-copy');

app.on('ready', () => {
  // Your code here
})
```

More info here: https://www.electronjs.org/docs/latest/api/command-line-switches
